opengrok-web/src/main/webapp/js/utils-0.0.41.js

```diff
@@ -1700,8 +1700,7 @@ function pageReadyMast() {
 }
 
 function domReadyMenu(minisearch) {
-    if (getCookie('OpenGrokSuggester.enabled') === 'false') {
-        console.log('Suggester disabled by a cookie');
```

Comment:
do we want to retain some sort of `console.log()` ?

Reply:
I think it's a good javascript practice to remove `console.log()` calls from production code. I quickly checked if google closure could not be configured to do that but it does not seem so. Although this hack could be used: https://stackoverflow.com/questions/2934509/exclude-debug-javascript-code-during-minification#answer-5969752

But I can add it back if you think the logs are beneficial as I do not have a strong opinion about it.

Reply:
I am fine with whatever the current best practice is.